components/accelerator/nvidia/sxid/component.go

```diff
@@ -217,7 +219,7 @@ func (c *SXIDComponent) SetHealthy() error {
 
 func (c *SXIDComponent) updateCurrentState() error {
 	var rebootErr string
-	rebootEvents, err := getRebootEvents(c.rootCtx)
+	rebootEvents, err := reboot.GetEvents(c.rootCtx, c.eventStore, "os", time.Now().Add(-DefaultRetentionPeriod))
```

Comment:
The caller should not know the `os`, even it should not pass the eventstore to the reboot package.. we can create a global reboot interface object like eventstore.. some static information in os component should be initialized once, and pass them for each component which depends on them.